lib/widgets/molecules/profile_experience_and_education.dart

```diff
@@ -47,21 +50,37 @@ class _Experience extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final theme = Theme.of(context);
+    final l10n = AppLocalizations.of(context)!;
     final items = _exampleExperience
-        .map((exp) => [
-              Text(exp.position, style: theme.textTheme.labelMedium),
-              Text(exp.companyName),
-              Text("(${exp.dateRange}) · ${exp.timeRange}"),
-              Text(exp.location),
-              const SizedBox(height: Insets.paddingMedium),
-            ])
+        .map((exp) {
+          final companyName = (exp.companyUrl == null)
+              ? Text(exp.companyName)
+              // https://stackoverflow.com/a/54540261
+              : InkWell(
+                  onTap: () => launchUrl(Uri.parse(exp.companyUrl ?? "")),
+                  child: Text(
+                    exp.companyName,
+                    style: const TextStyle(
+                        decoration: TextDecoration.underline,
+                        color: Colors.blue),
+                  ),
+                );
```

Comment:
The design has the website as a separate line at the bottom instead of turning the company into a hyperlink.

Reply:
Fixed.